applications/fax/src/fax_worker.erl

```diff
@@ -939,7 +939,13 @@ fetch_document_from_url(JObj) ->
 prepare_contents(JobId, RespHeaders, RespContent) ->
     lager:debug("preparing fax contents", []),
     TmpDir = kapps_config:get_binary(?CONFIG_CAT, <<"file_cache_path">>, <<"/tmp/">>),
-    case fax_util:normalize_content_type(props:get_value("content-type", RespHeaders, <<"application/octet-stream">>)) of
+    ContentType = props:get_first_defined(["content-type"
+                                          ,"Content-Type"
```

Comment:
wrong line. i think the fix should be in line 917 `Props = [{"Content-Type", ContentType}],`  to `Props = [{"content-type", ContentType}],`